source/_patterns/00-atoms/global/colors.mustache

```diff
@@ -1,8 +1,67 @@
 <ul class="sg-colors">
 
+	<h2>Primary Theme Colors</h2>
+
+	<li>
+		<span class="sg-swatch bg--emperor" title="" style="background: "></span>
+		<span class="sg-label small">Emperor</span>
```

Comment:
@SRHannah18 To get smaller type here, try using one of the type styles like `font--secondary--xs` or `font--secondary--xs` so it can fit in the area. OR, you can make those swatches a little wider. They don't have to abide by the PL example swatches.